homeassistant/components/buienradar/weather.py

```diff
@@ -150,25 +152,33 @@ def temperature_unit(self):
     @property
     def forecast(self):
         """Return the forecast array."""
-        from buienradar.buienradar import (CONDITION, CONDCODE, RAIN, DATETIME,
-                                           MIN_TEMP, MAX_TEMP)
-
-        if self._forecast:
-            fcdata_out = []
-            cond = self.hass.data[DATA_CONDITION]
-            if self._data.forecast:
-                for data_in in self._data.forecast:
-                    # remap keys from external library to
-                    # keys understood by the weather component:
-                    data_out = {}
-                    condcode = data_in.get(CONDITION, []).get(CONDCODE)
-
-                    data_out[ATTR_FORECAST_TIME] = data_in.get(DATETIME)
-                    data_out[ATTR_FORECAST_CONDITION] = cond[condcode]
-                    data_out[ATTR_FORECAST_TEMP_LOW] = data_in.get(MIN_TEMP)
-                    data_out[ATTR_FORECAST_TEMP] = data_in.get(MAX_TEMP)
-                    data_out[ATTR_FORECAST_PRECIPITATION] = data_in.get(RAIN)
-
-                    fcdata_out.append(data_out)
-
-            return fcdata_out
+        from buienradar.constants import (CONDITION, CONDCODE, RAIN, DATETIME,
+                                          MIN_TEMP, MAX_TEMP, WINDAZIMUTH,
+                                          WINDSPEED)
+
+        if not self._forecast:
+            return
```

Comment:
```suggestion
            return None
```

Explicit None return if other returns return a value.